CodeTheDream/ride-share-app
# removed the ride complete button

## Description

What does the PR do?
Removes the unnecessary complete button in the RideView page.

## Background Information
Complete ride button is redundant since dropping off the rider automatically completes the ride and we don't want to say the ride is completed unless the rider is dropped off.

## Screenshots
![Screen Shot 2021-02-11 at 23 43 56](https://user-images.githubusercontent.com/63831504/107792843-36734280-6d1b-11eb-9188-ca1c443219fa.png)

## Checklist:
- [v] Screenshots attached (if dealing with UI)

## Testing Steps for reviewer:
- Step 1 
Go to RideView.js and see the lines 269-289 and 554-560 being commented out. 
- Step 2
Check if the modal doesn't show the complete ride button on your simulator. (it should now show only the ride cancel button.)
